tests/gcp/hooks/test_bigquery.py

```diff
@@ -71,66 +71,6 @@ def test_credentials_provided(self, mock_read_gbq, _):
         self.assertEqual("PROJECT_ID", kwargs['project_id'])
 
 
-class TestBigQueryDataframeResults(unittest.TestCase):
```

Comment:
It looks like an unrelated change. Can you remove it from this PR?

Reply:
Sure, i've deleted.